tests/unit/store/createSubcollectionStore.ts

```diff
@@ -1,8 +1,24 @@
 import * as registerSuite from 'intern!object';
 import * as assert from 'intern/chai!assert';
-import { createData, ItemType, patches } from '../support/createData';
+import { createData, ItemType, patches, createUpdates } from '../support/createData';
 import { CrudOptions } from '../../../src/store/createStore';
 import createSubcollectionStore from '../../../src/store/createSubcollectionStore';
+import createAsyncStorage from '../support/AsyncStorage';
+
+function getStoreAndDfd(test: any) {
+	const dfd = test.async(1000);
+	const store = createSubcollectionStore<ItemType, CrudOptions>();
+	const Subcollection = store.createSubcollection();
```

Comment:
I actually had the same question. The `getStoreAndDfd()` here [is pulled up](https://github.com/dojo/stores/pull/31/files#diff-781607b689cef84922673aa9bbfce8dbL20) for better reusability and consistency. Maybe @maier49 can let us know.